devdoc/umock_c_lib_requirements_with_ids.md

```diff
@@ -893,6 +893,39 @@ TEST_FUNCTION(validate_argument_as_type_sample)
 }
 ```
 
+### CaptureArgumentValue_{arg_name}(arg_type* arg_value)
+
+**SRS_UMOCK_C_LIB_01_209: [** The `CaptureArgumentValue_{arg_name}` shall copy the value of the argument at the time of the call to `arg_value`. **]**
```

Comment:
>copy [](start = 74, length = 4)

would be nice if the documentation would specify "how" does it copy it: is it a bitwise-copy, or smarter somehow by using some copy-constructors defined by the user.